src-java/grpc-speaker/grpc-service/src/main/java/org/openkilda/grpc/speaker/mapper/NoviflowResponseMapper.java

```diff
@@ -44,4 +47,23 @@
     SwitchEthLinkInfoStatus toSwitchEthLink(StatusSwitchEthLink statusSwitchEthLink);
 
     SwitchBuildInfoStatus toSwitchBuildInfo(StatusSwitchBuild statusSwitchBuild);
+
+    PacketInOutStatsResponse toPacketInOutStatsResponse(PacketInOutStats stats);
+
+    /**
+     * Maps YesNo enum to Boolean value.
+     */
+    default Boolean toBoolean(YesNo yesNo) {
+        if (yesNo == null) {
+            return null;
+        }
+        switch (yesNo) {
+            case YES:
+                return true;
+            case NO:
+                return false;
+            default:
+                return null;
```

Comment:
it slightly overlaps with line 56 check

Reply:
We can't remove check in line 57 because it will cause NPE in line 60.
Also we can't remove `default` section because:
1. stylecheck will raise a warning
2. YesNo enum has other values like `UNRECOGNIZED` and `YESNO_RESERVED`. We must handle them